### ISSUE TITLE
orderColumn not works in Datatables as Service

### ISSUE THREAD

**Issue by fanjavaid** [yajra/laravel-datatables#312]
Hello, i can't use orderColumn in my datatables.
Here is my implementation.
I want to Descending order by created_at column.

```php
public function ajax()
    {
        return $this->datatables
            ->of($this->query())
            ->orderColumn('penyedia_paket_satker.created_at', 'name $1')
            ->addColumn('action', function ($data) {
                return '
                    <a href="' . route('adm1n.rekapitulasi-rup.show', $data->id) . '" class="btn btn-xs btn-primary"><i class="fa fa-eye"></i></a>
                ';
            })
            ->make(true);
    }
```

Thank you ^^

**Comment by yajra** [comment]
Will try to check this. BTW, you can also achieve the same result by using `orderData` of datatables js. See this for ref: https://datatables.net/reference/option/columns.orderData

**Comment by fanjavaid** [comment]
Hello i have same issue an apply your solution like this :
```php
[
                    'title' => 'ID Paket',
                    'data' => 'id_paket',
                    'name' => 'id_paket',
                    'visible' => true,
                    'orderData' => 1
                ],
```
But it won't work
I have this data id as a big number 5250421 but in my sort just show 660169
And my ID Paket column at the first one.

I'm confused what method that i must use, orderColumn or orderData property.
Both not work.

**Comment by ChaosPower** [comment]
@fanjavaid @yajra said you should try by doing the datatables (**javascript**) approach. You did it using php.

**Comment by yajra** [comment]
@fanjavaid, I just released v6.6.0 which basically improves column name detection of the package. Can you test and confirm if this issue is fixed? Thanks!

**Comment by yajra** [comment]
I just tested `orderColumn` on latests version and it works for me.
